lib/decompress.ps1

```diff
@@ -104,6 +127,68 @@ function Expand-7zipArchive {
     }
 }
 
+function Expand-7zip-zstdArchive {
```

Comment:
- Why did you duplicate whole `Expand-7zipArchive` function?
	- Just add `-Zstd` parameter to `Expand-7zipArchive`.
- `Expand-7zip-zstdArchive` is horrible name. Never use hyphens in powershell noun part of command.